src/composables/useLeaflet.ts

```diff
@@ -38,9 +38,11 @@ export const useLeaflet = (
     // internal refs
     const geoJsonLayer = shallowRef<GeoJSON<FeatureProperties, Geometry>>(geoJSON(undefined));
     const countryOutlineLayer = shallowRef<Polyline | null>(null);
+    const selectedAdmin1OutlineLayer = shallowRef<(Polyline | null)[] | null>(null);
```

Comment:
this variable name implies to me that the user has selected an admin1 region - but you mean they've selected a country.

Reply:
yep gone with `countryAdmin1OutlineLayer` instead